wxgtk2 dependency issue

as per [Antonio Rojas's post on news](https://archlinux.org/news/wxwidgets-32-update-may-need-manual-intervention/), wxgtk is being renamed wxwidgets and dropping gtk2 support. urbackup2-client needs dependencies adjusted.

Well, removing wxgtk2 and following instructions already find another dependency correctly (as of today on a Manjaro installation, if you chose wxwidgets2 as replacement for wxgtk2); but build phase still breaks with the following error:
```
client/main.cpp: In member function ‘virtual bool MyApp::OnInit()’:
client/main.cpp:354:9: error: duplicate case value
  354 |         case wxLANGUAGE_SPANISH_MODERN:
      |         ^~~~
client/main.cpp:342:9: note: previously used here
  342 |         case wxLANGUAGE_SPANISH:
      |         ^~~~
client/main.cpp:375:9: error: duplicate case value
  375 |         case wxLANGUAGE_CHINESE_TAIWAN:
      |         ^~~~
client/main.cpp:374:9: note: previously used here
  374 |         case wxLANGUAGE_CHINESE_TRADITIONAL:
      |         ^~~~
make[2]: *** [Makefile:4564: client/urbackupclientgui-main.o] Errore 1
```
I don't have sufficient knowledge to understand if this is a simple-to-fix syntax-related issue or much more deep in the sources, unfortunately, so I cannot provide a patch.

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,13 +2,13 @@
 # Ex-Maintainer: K0n24d <konrad AT knauber DOT net>
 pkgname=urbackup2-client
 pkgver=2.4.11
-pkgrel=3
+pkgrel=4
 pkgdesc="Client Server backup system"
 arch=('i686' 'x86_64' 'armv5' 'armv6h' 'armv6' 'armv7h' 'armv7' 'aarch64')
 url="http://www.urbackup.org/"
 license=("GPL")
 makedepends=('gcc-libs' 'gcc' 'make')
-depends=('wxgtk2' 'crypto++' 'zlib')
+depends=('wxwidgets' 'crypto++' 'zlib')
 conflicts=('urbackup2-client-no-gui' 'urbackup-client-no-gui' 'urbackup-client')
 source=(
     "https://www.urbackup.org/downloads/Client/${pkgver}/urbackup-client-${pkgver}.0.tar.gz"
@@ -20,6 +20,7 @@ source=(
     'lvm_create_filesystem_snapshot'
     'lvm_remove_filesystem_snapshot'
     'md5-bytes.patch'
+    'double_language.patch'
 )
 sha512sums=('a0cae5dad805d11e2764b2635dd37a4e24e0d029bae1eefc471f2b87262ac2448b3d123c2ffb5deb2eccab1baed0e31d3711e8837d66b93edf736fb113145ef1'
             '416fb8f5f3687a3c369cc2b199d4c8b4170494f0a119566a91ac6a0c2f202dc5049804c10508b66ba657011b39be5ddd055091cd531a665b4398899f404086ca'
@@ -29,7 +30,8 @@ sha512sums=('a0cae5dad805d11e2764b2635dd37a4e24e0d029bae1eefc471f2b87262ac2448b3
             '238c286d451474a8721292f7e98b4f13600cb430c16a27ceb9551cc83705b8268a3f1202785fb5b61523f372b4e7e804fd20b7db62677621983d79a271aa106b'
             'a2d4ba03ae15582d2cd74ff68c38ff0f90d75a6eb5c241f9a022b0652fa2dc9b184439f6bda9a9538645925f739503ee7b3fc7bb232589583cdeb6dc27d74e5c'
             '9bdfefccdd9d6e37a77975324a7c417f3de2aa59e6da0bfde3c318b8c6f3d7f4629f3a41eebee548b9c572b8ed39640434cc08bd020d25362fddffc4426438de'
-            '34e25c868cf4572414fbc6c693877127152f9a97edf8865b4263a55cf16f71a5045ba96b1a9af8244ed49c35cab56e3fdb44348d191e9f85e2efb66392907132')
+            '34e25c868cf4572414fbc6c693877127152f9a97edf8865b4263a55cf16f71a5045ba96b1a9af8244ed49c35cab56e3fdb44348d191e9f85e2efb66392907132'
+            'fb6c43d725d4ea201bdf91b1482473f205834c14f24906041d5c6ef22b1e0f4cec16d5e765a3fda6fed4d3b98f6217b190dd5d1e84051525205e22747c5eefac')
 
 CFLAGS="-march=native -O2 -pipe -fstack-protector-strong"
 CXXFLAGS="${CFLAGS} -ansi -std=gnu++11"
@@ -40,6 +42,7 @@ build() {
     sed  -i '/\#include \"cryptopp_inc.h\"/a #include "assert.h"' "${srcdir}/urbackup-client-${pkgver}.0/cryptoplugin/AESGCMDecryption.h"
 
     patch -d"${srcdir}/urbackup-client-${pkgver}.0" -p0 < "${srcdir}/md5-bytes.patch"
+    patch --forward --strip=1 --input="${srcdir}/double_language.patch"
 
     cd "${srcdir}/urbackup-client-${pkgver}.0"
     ./configure --prefix=/usr --sbindir=/usr/bin --localstatedir=/var --sysconfdir=/etc --enable-embedded-cryptopp

--- a/double_language.patch
+++ b/double_language.patch
@@ -1,0 +1,28 @@
+diff '--color=auto' --unified --recursive --text src_ori/urbackup-client-2.4.11.0/client/main.cpp src_neu/urbackup-client-2.4.11.0/client/main.cpp
+--- src_ori/urbackup-client-2.4.11.0/client/main.cpp	2020-12-04 21:46:10.000000000 +0100
++++ src_neu/urbackup-client-2.4.11.0/client/main.cpp	2022-07-30 13:06:26.313191211 +0200
+@@ -351,7 +351,7 @@
+ 	case wxLANGUAGE_SPANISH_GUATEMALA:
+ 	case wxLANGUAGE_SPANISH_HONDURAS:
+ 	case wxLANGUAGE_SPANISH_MEXICAN:
+-	case wxLANGUAGE_SPANISH_MODERN:
++    //case wxLANGUAGE_SPANISH_MODERN:
+ 	case wxLANGUAGE_SPANISH_NICARAGUA:
+ 	case wxLANGUAGE_SPANISH_PANAMA:
+ 	case wxLANGUAGE_SPANISH_PARAGUAY:
+@@ -371,7 +371,7 @@
+ 		lang= sysdef;
+ 		g_lang="zh_CN";
+ 		break;
+-	case wxLANGUAGE_CHINESE_TRADITIONAL:
++    //case wxLANGUAGE_CHINESE_TRADITIONAL:
+ 	case wxLANGUAGE_CHINESE_TAIWAN:
+ 		lang= sysdef;
+ 		g_lang="zh_TW";
+@@ -994,4 +994,4 @@
+ }
+ #else
+ IMPLEMENT_APP(MyApp)
+-#endif
+\ Kein Zeilenumbruch am Dateiende.
++#endif

